index.html

```diff
@@ -62,6 +62,157 @@ <h1>
 
         </div>
     </section>
+    <!-- Works -->
+    <section class="works">
+        <div class="works_content">
+            <h1>My Recent Works</h1>
+            <img src="Assets/Indicator.png" alt="">
+        </div>
+        <div class="works_cards">
+            <div class="card">
+                <div class="card_image">
+                    <img src="" alt="">
+                </div>
```

Comment:
- [ ] To enhance the visual appeal of your page and make your project more captivating to potential recruiters,
- [ ] Please🙏🙏 ensure that you utilize card images that align with the Figma design. This will contribute to an overall polished and professional appearance. 
- [ ] Additionally, it is important to include alternative text (alt text) for your images to ensure accessibility for all users.🥱🥱